### PR TITLE
Changes for OpenShift v4 / oidc

### PR DESCRIPTION
**Pull request by ozRunAI**

### CODE REVIEW

File: docs/admin/runai-setup/authentication/sso.md
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/sso.md:32]: @ozRunAI no need for a "-" when useing the mdash code. "-" is an "en-dash". 
```suggestion
* **Base URL**&mdash;the OpenShift API Server endpoint (example: https://api.<cluster-url>:6443)
```

File: docs/admin/runai-setup/authentication/sso.md
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/sso.md:33]:
```suggestion
* **ClientID**&mdash;the ID used to identify the client with the Authorization Server.
```

File: docs/admin/runai-setup/authentication/sso.md
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/sso.md:89]:
```suggestion
	4. Copy the `Redirect URL` and use it in your identity provider.
	5. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
        6. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
        7.When complete, press `Save`.
```
@ozRunAI Please be careful of the spaces you add. If these are meant to be indented ordered list, then you need to add spaces in the lines above before you indent them.

File: docs/admin/runai-setup/authentication/sso.md
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/sso.md:107]:
```suggestion
	4. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional). Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
        5. In the `User attributes` field enter the attribute and the value in the identity provider (optional).
        6. When complete, press `Save`.
```

**Comment by ozRunAI** [comment]
@jasonnovichRunAI please do not merge until @talevy-runai approves the PR

**Review by jasonnovichRunAI — COMMENTED** [review]
@ozRunAI I made some changes - please review so that for the future the markdown will look right. 8-)

**Review by talevy-runai — APPROVED** [review]

**Review by jasonnovichRunAI — APPROVED** [review]

**Pull request merged**
